src/main/java/org/tarantool/jdbc/ds/SQLDataSource.java

```diff
@@ -0,0 +1,163 @@
+package org.tarantool.jdbc.ds;
+
+import org.tarantool.jdbc.SQLConnection;
+import org.tarantool.jdbc.SQLProperty;
+
+import java.io.PrintWriter;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.SQLFeatureNotSupportedException;
+import java.sql.SQLNonTransientException;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Logger;
+import javax.sql.DataSource;
+
+/**
+ * Simple {@code java.sql.DataSource} implementation.
+ */
+public class SQLDataSource implements TarantoolDataSource, DataSource {
+
+    private PrintWriter logWriter;
+    private String description;
+    private String name;
```

Comment:
Should not we provide a default hostname (e.g. "localhost")? I see pgjdbc does.

Maybe applicable to logger too.

Reply:
The `name` related to datasourceName rather than `serverName`. PG provides that for the server name and doesn't support the data source name. 

Motivation to support this:
- suggested as a standard DS property in the spec;
- AFAIK app can name its data sources to distinguish them in application logic.

Reply:
> The `name` related to datasourceName rather than `serverName`.

Okay.

We however can provide default name / description. This can be usable for, say, logging.